.github/workflows/moodle-ci.yml

```diff
@@ -22,7 +22,7 @@ on:
   workflow_dispatch:
 
 jobs:
-  test:
+  test-legacy:
```

Comment:
Which is the reason to add a `test-legacy job` for legacy tests instead of a step on `test job`?